src/components/controls/atoms/TelegramEmbed.js

```diff
@@ -68,7 +68,7 @@ class TelegramEmbed extends Component {
     );
   }
 
-  componentWillReceiveProps({ src }) {
+  UNSAFE_componentWillReceiveProps({ src }) {
```

Comment:
This doesn't change the behavior and just shuts up the warning. Actually migrating away would involve more work and can be done separately.